get_started.sh

```diff
@@ -106,6 +106,12 @@ sudo ln -s -f /usr/share/clion*/bin/clion.sh /usr/local/bin/clion
 cd $DIR
 ./install_dependencies.sh
 
+##############################
+# Setup Snowbots Udev Rules  #
+##############################
+cd $DIR
+./setup_udev_rules.sh
```

Comment:
Doesn't look like `setup_udev_rules.sh` is part of this pull request?

Reply:
Also, we should put the script with the rest of the firmware stuff in `src/firmware`. We'd also have to change the path here to: `./src/firmware/setup_udev_rules.sh`